lib/dealership.rb

```diff
@@ -0,0 +1,44 @@
+require 'pry'
+class Dealership
+  attr_reader :inventory
+  def initialize(argument_1, argument_2)
+    @argument_1 = argument_1
+    @argument_2 = argument_2
+    @inventory = []
+  end
```

Comment:
I might rename `argument_1` and `argument_2` to `name` and `address`
```suggestion
  def initialize(name, address)
    @name = name
    @address = address
    @inventory = []
  end
```